homestar-runtime/tests/utils.rs

```diff
@@ -221,6 +225,23 @@ pub(crate) fn kill_homestar_daemon() -> Result<()> {
     Ok(())
 }
 
+/// Remove sqlite database and associated temporary files
+pub(crate) fn remove_db(name: &str) {
+    let _ = fs::remove_file(format!("{name}"));
+    let _ = fs::remove_file(format!("{name}-shm"));
+    let _ = fs::remove_file(format!("{name}-wal"));
+}
+
+/// Wait for socket connection or timeout
+pub(crate) fn wait_for_socket_connection(port: u16, exp_retry_base: u64) -> Result<(), ()> {
```

Comment:
love this as a util. Update the other tests (in a quickie follow-up PR if you want)?

Reply:
Yep! Will do. 💯